renderer/components/Onboarding/Steps/SeedWordList.js

```diff
@@ -0,0 +1,103 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import bip39 from 'bip39-en'
+import { Flex } from 'rebass/styled-components'
+import styled from 'styled-components'
+import { themeGet } from '@styled-system/theme-get'
+import { Text } from 'components/UI'
+
+/**
+ * Create a mapping of first letter to words in BIP 39 list so
+ * we don't have to filter through the whole array when looking up values
+ * every time
+ */
+const bip39WordMap = bip39.reduce((bip39WordMap, word) => {
+  const [firstLetter] = word
+  if (!bip39WordMap[firstLetter]) {
+    bip39WordMap[firstLetter] = []
+  }
+
+  bip39WordMap[firstLetter].push(word)
+
+  return bip39WordMap
+}, {})
+
+const StyledOptionList = styled.ul`
```

Comment:
Generally speaking, we prefer to use rebass over raw styled-components as this generally results in more flexible components that support all of the features of rebass.

This could be re-written something more like this:

```jsx
const StyledOptionList = React.forwardRef(({ ...rest }, ref) => (
  <Box
    ref={ref}
    as="ul"
    bg="secondaryColor"
    maxHeight={300}
    mt={1}
    p={0}
    sx={{
      position: 'absolute',
      zIndex: 2,
      overflowY: 'auto',
      overflowX: 'hidden',
      outline: 0,
      transition: 'opacity 0.1s ease',
      borderRadius: 's',
      boxShadow: 's',
    }}
    width={1}
    {...rest}
  />
))
StyledOptionList.displayName = 'StyledOptionList'
```

Reply:
Done.